--- kmonad-alex.kbd	2025-04-04 22:09:39.942189702 +0530
+++ kmonad-alex-internal.kbd	2025-04-04 22:12:40.763189013 +0530
@@ -16,7 +16,7 @@
   tab  q    w    e    r    t    y    u    i    o    p    [    ]    \   
   caps a    s    d    f    g    h    j    k    l    ;    '    ret      
   lsft z    x    c    v    b    n    m    ,    .    /         rsft          
-  lctl lmet lalt           spc            ralt   fn     cmp   rctl          
+  lctl lmet lalt           spc            ralt
 )
 
 (deflayer colemak-dh-cust
@@ -25,7 +25,7 @@
   tab   q    w    f    p    b    j    l    u    y    grv  @num  ;    lmet
   @msc  a    r    s    t    g    m    n    e    i    o    '     ret
   lsft  x    c    d    v    z    k    h    ,    .    /          rsft          
-  lctl  lalt lmet           spc            ralt   fn     rctl   ret
+  lctl  lalt lmet           spc            ralt
 )
 
 (deflayer qwerty
@@ -34,7 +34,7 @@
   tab   q    w    e    r    t    y    u    i    o    p    [    ]    \   
   @msc  a    s    d    f    g    h    j    k    l    ;    '    ret      
   lsft  z    x    c    v    b    n    m    ,    .    /         rsft          
-  lctl lmet lalt           spc             ralt   fn     cmp   rctl          
+  lctl lmet lalt           spc             ralt
 )
 
 (deflayer layoutswitch
@@ -43,7 +43,7 @@
   XX  XX   XX   XX  XX  XX  XX  XX  XX  XX  XX  XX  XX  XX 
   XX  XX   XX   XX  XX  XX  XX  XX  XX  XX  XX  XX  XX    
   XX  XX   XX   XX  XX  XX  XX  XX  XX  XX  XX      XX        
-  XX  XX   XX          XX           XX  XX  XX  XX        
+  XX  XX   XX          XX           XX
 )
 
 (defalias
@@ -61,7 +61,7 @@
   XX   7  8  9  0   XX  XX  XX  XX   XX    XX  XX  XX  XX 
   bspc 4  5  6  -   XX  XX  XX  lalt rsft  XX  XX  XX    
   spc  1  2  3  =   XX  XX  /   _    _     XX      rsft        
-  XX   XX  spc    lmet     XX  XX  XX   XX        
+  XX   XX  spc    lmet     XX
 )
 
 (defalias
@@ -73,7 +73,7 @@
   XX   XX XX    XX  XX  XX  lft down  up  rght  XX  XX  XX  XX  
   XX   XX lctl  XX  XX  XX  h   j     k   l     XX  XX  lmet
   lsft XX XX    XX  XX  XX  XX  XX    XX  XX    XX      spc
-  _    _   _        lsft       _     _   _      _       
+  _    _   _        lsft       _
 )
 
 (defalias
